gandalf.js

```diff
@@ -1,61 +1,64 @@
-const objectValues = Object.values(object);
```

Comment:
SCRIPT-This hardcoded value might be better suited as a constant at the top of the file.